### PR TITLE
report items constant to reuse

### PR DESCRIPTION
**Pull request by hotcakedev628**
We could reuse report items for the nav list, top menu and others.
I used the constant to render nav list in reports page.
https://github.com/CruGlobal/mpdx-react/blob/e2004b31ec3ca6a500278f6af77c5fd138773107/src/components/Reports/ReportLayout/NavReportsList/NavReportsList.tsx#L89-L95

### CODE REVIEW

File: src/components/Reports/ReportLayout/NavReportsList/ReportItems.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+export default [
```
Comment (OzzieOrca) [inline src/components/Reports/ReportLayout/NavReportsList/ReportItems.js:2]:
```suggestion
export const ReportNavItems = [
```

**Review by OzzieOrca — CHANGES_REQUESTED** [review]
Ya this looks like a good strategy. Once you merge your other PR, feel free to refactor both menus to use this file.

Can you make it a `.ts` file too?

**Comment by hotcakedev628** [comment]
@OzzieOrca Sure thing. But I just wondered if I could do it since the app didn't import any `.ts` files.

**Comment by OzzieOrca** [comment]
It should work. There's some `.ts` files around.

**Comment by hotcakedev628** [comment]
@OzzieOrca Can I merge this one?

File: src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx [hunk 2]
```diff
@@ -129,80 +130,25 @@ const NavMenu = (): ReactElement => {
...
+                            key={reportItem.id}
+                            href={`/accountLists/${state.accountListId}/reports/${reportItem.id}`}
+                            scroll={false}
+                          >
+                            <MenuItem onClick={handleReportsMenuClose}>
+                              <ListItemText
+                                primary={t(
+                                  `${reportItem.title}${
+                                    reportItem.subTitle
+                                      ? ' (' + reportItem.subTitle + ')'
```
Comment (OzzieOrca) [inline src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx:144]:
```suggestion
                                      ? ` (${reportItem.subTitle})`
```

**Review by OzzieOrca — APPROVED** [review]

**Review by OzzieOrca — APPROVED** [review]

**Comment by OzzieOrca** [comment]
Looks like you've got one failing test.

**Review by OzzieOrca — APPROVED** [review]

**Pull request merged**